CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.3)
+
+project(minmea)
+
+find_package(Threads REQUIRED) # Workaround for https://github.com/libcheck/check/issues/48#issuecomment-322965461
+find_package(PkgConfig)
+pkg_check_modules(CHECK REQUIRED check)
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -Wall -Wextra -Werror -std=c99")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -D_POSIX_C_SOURCE=199309L -D_BSD_SOURCE -D_DEFAULT_SOURCE -D_DARWIN_C_SOURCE")
+
+set(minmea_SRCS minmea.c minmea.h)
```

Comment:
only comment is that I'd probably group together the library lines, the example lines and the test lines but this is fine